forms/static/forms/admin/dependent_fields.js

```diff
@@ -49,63 +36,84 @@
         $('.field-special_qn_3').hide();
         $('.field-item_number').hide();
         $('.field-webpage_layer_no').hide();
-        $('.field-age').hide();
-        special_tabs($('.field-age'));
+        // For TV or Internet, do not hide age field
+        if(!(($('#televisionsheet_form').length > 0) || ($('#internetnewssheet_form').length > 0))) {
+            $('.field-age').hide();
+        }
     }
 
     $(document).ready(function() {
-        const monitor_mode = $("#id_monitor_mode").find(":selected").text().toLowerCase()
-        if(monitor_mode === "short"){
-            short_monitoring_mode()
+        // Remove "None" option from these fields
+        var person_mentioned_in_article = 0;
+        while($(`#id_newspaperperson_set-${person_mentioned_in_article}-victim_or_survivor_1`).length){
+            $(`#id_newspaperperson_set-${person_mentioned_in_article}-family_role li:first-child`).remove()
+            $(`#id_radioperson_set-${person_mentioned_in_article}-family_role li:first-child`).remove()
+            $(`#id_internetperson_set-${person_mentioned_in_article}-family_role li:first-child`).remove()
+            $(`#id_twitterperson_set-${person_mentioned_in_article}-family_role li:first-child`).remove()
+
+            $(`#id_newspaperperson_set-${person_mentioned_in_article}-victim_or_survivor li:first-child`).remove()
+            $(`#id_radioperson_set-${person_mentioned_in_article}-victim_or_survivor li:first-child`).remove()
+            $(`#id_internetperson_set-${person_mentioned_in_article}-victim_or_survivor li:first-child`).remove()
+            $(`#id_twitterperson_set-${person_mentioned_in_article}-victim_or_survivor li:first-child`).remove()
+
+            $(`#id_newspaperperson_set-${person_mentioned_in_article}-is_quoted li:first-child`).remove()
+            $(`#id_radioperson_set-${person_mentioned_in_article}-is_quoted li:first-child`).remove()
+            $(`#id_internetperson_set-${person_mentioned_in_article}-is_quoted li:first-child`).remove()
+            $(`#id_twitterperson_set-${person_mentioned_in_article}-is_quoted li:first-child`).remove()
+            person_mentioned_in_article+=1;
+        }
```

Comment:
Since we're checking `#id_newspaperperson` in the `while` loop, this loop won't run if we're in other type of sheets... We can take advantage of jQuery select id start/end to achieve the same thing without loops:
```suggestion
        $("ul[id^='id_newspaperperson_set-'][id$='-family_role'] li:first-child").remove();
        $("ul[id^='id_newspaperperson_set-'][id$='-victim_or_survivor'] li:first-child").remove();
        $("ul[id^='id_newspaperperson_set-'][id$='-is_quoted'] li:first-child").remove();
        
        $("ul[id^='id_radioperson_set-'][id$='-family_role'] li:first-child").remove();
        $("ul[id^='id_radioperson_set-'][id$='-victim_or_survivor'] li:first-child").remove();
        $("ul[id^='id_radioperson_set-'][id$='-is_quoted'] li:first-child").remove();
        
        $("ul[id^='id_televisionperson_set-'][id$='-family_role'] li:first-child").remove();
        $("ul[id^='id_televisionperson_set-'][id$='-victim_or_survivor'] li:first-child").remove();
        $("ul[id^='id_televisionperson_set-'][id$='-is_quoted'] li:first-child").remove();

        $("ul[id^='id_internetnewsperson_set-'][id$='-family_role'] li:first-child").remove();
        $("ul[id^='id_internetnewsperson_set-'][id$='-victim_or_survivor'] li:first-child").remove();
        $("ul[id^='id_internetnewsperson_set-'][id$='-is_quoted'] li:first-child").remove();

        $("ul[id^='id_twitterperson_set-'][id$='-family_role'] li:first-child").remove();
        $("ul[id^='id_twitterperson_set-'][id$='-victim_or_survivor'] li:first-child").remove();
        $("ul[id^='id_twitterperson_set-'][id$='-is_quoted'] li:first-child").remove();
```

Reply:
> Since we're checking `#id_newspaperperson` in the `while` loop, this loop won't run if we're in other type of sheets... We can take advantage of jQuery select id start/end to achieve the same thing without loops:

I had just pushed something like a second before this comment. Not sure if that will work.